### DISCUSSION TITLE
Suggestion - Add Calcified Rocks to Mining plugin

### DISCUSSION THREAD

**Discussion by arch625** [runelite/runelite#17938] [Ideas]
There is currently a Cam Torum Mine plugin that highlights only the water streams, however it would be nice if the Mining plugin highlighted the Calcified Rocks. 

https://oldschool.runescape.wiki/w/Calcified_rocks

**Comment by YvesW** [comment]
Iirc the author of that plugin started working on a core PR, but it's not finished yet. https://github.com/runelite/runelite/pull/17728
